src/assets/styles/colors.js

```diff
@@ -16,6 +16,12 @@ export const ivoryDark = '#FFFEF0';
 export const black = '#161616';
 export const transparent = 'transparent';
 
```

Comment:
let's kill these additional colors. let's go with primary and secondary only for now.